Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:16
```

Comment:
You think its better to have the other Dockerfile with version 18?

Reply:
Whatever you think is best - I figured you would update that file as you want.  I'm not at all familiar with dockerfiles, so it is really hard for me to tell what to include.